src/pages/articles.tsx

```diff
@@ -79,18 +88,33 @@ function getNewTags(currentFilterTags, tag) {
   return [...currentFilterTags, tag]
 }
 
-const Articles = ({ data: { site, allMdx } }) => {
+export interface EdgeType {
+  node: {
+    id: string;
+    fields: {
+      slug: string;
+      tags: string[];
+    }
+    frontmatter: {
+        title: string;
+        slug: string;
+    };
+    excerpt: string;
+  };
+}
+
+const ArticlesPage = ({ data: { site, allMdx } }: PageProps<ArticlesPageQuery>) => {
```

Comment:
All pages use `PageProps<BlaBlaQuery>` which was generated with `graphql-codegen`

I didn't configure running it after each change though (you need to do it manually)

If needed, I can add it in the next MR

WDYT?

Reply:
I don't mind doing it manually! that should be fun. I've looked at `graphql-codegen` but don't know if i've used it. i'm sure it's easy to figure out